strings/br.py

```diff
@@ -0,0 +1,433 @@
+                       "Se você deseja iniciar um novo, envie um novo comando / start."
+
+# User menu: order
+menu_order = "🛒 Order de Produtos"
+
+# User menu: order status
+menu_order_status = "🛍 Minhas orders"
+
+# User menu: add credit
+menu_add_credit = "💵 Adicionar fundos"
+
+# User menu: bot info
+menu_bot_info = "ℹ️ Bot info"
+
+# User menu: cash
+menu_cash = "💵 Com dinheiro"
+
+# User menu: credit card
+menu_credit_card = "💳 Com Cartão de Crédito"
+
+# Admin menu: products
+menu_products = "📝️ Produtos"
+
+# Admin menu: orders
+menu_orders = "📦 Orders"
+
+# Menu: transactions
+menu_transactions = "💳 Lista de Transações"
+
+# Menu: edit credit
+menu_edit_credit = "💰 Criar transação"
+
+# Admin menu: go to user mode
+menu_user_mode = "👤 Mudar para o modo de cliente"
+
+# Admin menu: add product
+menu_add_product = "✨ Novo Produto"
+
+# Admin menu: delete product
+menu_delete_product = "❌ Apagar produto"
+
+# Menu: cancel
+menu_cancel = "🔙 Cancelar"
+
+# Menu: skip
+menu_skip = "⏭ Pular"
+
+# Menu: done
+menu_done = "✅️ Concluido"
+
+# Menu: pay invoice
+menu_pay = "💳 Pagar"
+
+# Menu: complete
+menu_complete = "✅ Complete"
+
+# Menu: refund
+menu_refund = "✴️ Devolver"
+
+# Menu: stop
+menu_stop = "🛑 Parar"
+
+# Menu: add to cart
+menu_add_to_cart = "➕ Add"
+
+# Menu: remove from cart
+menu_remove_from_cart = "➖ Remover"
+
+# Menu: help menu
+menu_help = "❓ Ajuda / Suporte"
+
+# Menu: guide
+menu_guide = "📖 Guia"
+
+# Menu: next page
+menu_next = "▶️ Proximo"
+
+# Menu: previous page
+menu_previous = "◀️ Anterior"
+
+# Menu: contact the shopkeeper
+menu_contact_shopkeeper = "👨‍💼 Contatar a loja"
+
+# Menu: generate transactions .csv file
+menu_csv = "📄 .csv"
+
+# Menu: edit admins list
+menu_edit_admins = "🏵 Editar Gerentes"
+
+# Menu: language
+menu_language = "🇬🇧 Linguagem"
+
+# Emoji: unprocessed order
+emoji_not_processed = "*️⃣"
+
+# Emoji: completed order
+emoji_completed = "✅"
+
+# Emoji: refunded order
+emoji_refunded = "✴️"
+
+# Emoji: yes
+emoji_yes = "✅"
+
+# Emoji: no
+emoji_no = "🚫"
+
+# Text: unprocessed order
+text_not_processed = "pendente"
+
+# Text: completed order
+text_completed = "completo"
+
+# Text: refunded order
+text_refunded = "devolvido"
+
+# Add product: name?
+ask_product_name = "Qual deve ser o nome do produto?"
+
+# Add product: description?
+ask_product_description = "Qual deve ser a descrição do produto?"
+
+# Add product: price?
+ask_product_price = "Qual deve ser o preço do produto?\n" \
+                    "Entre <code>X</code> se ainda não quiser que o produto esteja à venda."
+
+# Add product: image?
+ask_product_image = "🖼 Que imagem você quer que o produto tenha?\n" \
+                    "\n" \
+                    "<i>Envie a foto ou ignore esta fase e não adicione nenhuma imagem.</i>"
+
+# Order product: notes?
+ask_order_notes = "Gostaria de deixar um recado junto com o pedido?\n" \
+                  "💼 Ele ficará visível para os gerentes da loja.\n" \
```

Comment:
```suggestion
                  "Ele ficará visível para os 💼 Gerentes da loja.\n" \
```
I think the emoji should be here, but i'm not sure.